### PR TITLE
remove forest-admin

### LINKED ISSUE #323: Uninstall Forest Admin
# Feature
## Why is this feature being added?

This PR added Forest admin: https://github.com/OperationCode/operationcode_backend/pull/283

It needs to be uninstalled, as we are using ActiveAdmin instead.  Here are the instructions to uninstall forestadmin:

https://doc.forestadmin.com/knowledge-base.html#uninstall
## What should your feature do?

- [ ] Clean uninstall of ForestAdmin and everything from https://github.com/OperationCode/operationcode_backend/pull/283

### PR DESCRIPTION
**Pull request by kirbygit**
# Description of changes

- Completely uninstall gem 'forest_liana' dependencies
- removed Forest routes in config/routes.rb
- removed forest_liana.rb in initializers
- run `make bundle` to update dependencies

# Issue Resolved

Fixes #323

### CODE REVIEW

**Comment by kirbygit** [comment]
What does this mean?

`Failure:
GitHub::AuthenticationTest#test_#set_options_merges_the_authenticated_options_hash_when_Rails.env.prod?_is_true [/app/test/lib/git_hub/authentication_test.rb:35]:
Expected false to be truthy.`

**Comment by kirbygit** [comment]
`
// I tried to use assert_equal on this test
assert_equal @instance.set_options == {
      :query =>
        {
          :client_id => "some random id",
          :client_secret => "some random key",
          :per_page => 100
        },
      :headers =>
        {
          "Accepts" => "application/vnd.github.v3+json",
          "User-Agent" => "operationcode"
        }
      }`

**Comment by hpjaj** [comment]
Looks like [your `secrets.yml` file](https://github.com/kirbygit/operationcode_backend/blob/uninstall-forest-admin-v2/config/secrets.yml) is missing some attrs.  For example all of the `git_hub` keys for all three envs :

https://github.com/OperationCode/operationcode_backend/blob/master/config/secrets.yml#L24-L25

You'll need to rebase on the OC master branch.

**Comment by kirbygit** [comment]
lol it took me awhile on this issue

File: config/secrets.yml [hunk 1]
```diff
@@ -31,12 +29,9 @@ test:
 # Do not keep production secrets in the repository,
 # instead read values from the environment.
 production:
-  forest_env_secret: <%= ENV["FOREST_ENV_SECRET"] %>
-  forest_auth_secret: <%= ENV["FOREST_AUTH_SECRET"] %>
-  git_hub_client_id: <%= ENV["GIT_HUB_CLIENT_ID"] %>
-  git_hub_client_secret: <%= ENV["GIT_HUB_CLIENT_SECRET"] %>
```
Comment (hpjaj) [inline config/secrets.yml]: These two GitHub production env vars actually remain.

Comment (hpjaj) [reply config/secrets.yml]: These Forest admin ones need to be removed:

```
   forest_env_secret: <%= ENV["FOREST_ENV_SECRET"] %>
   forest_auth_secret: <%= ENV["FOREST_AUTH_SECRET"] %>
```

The GitHub ones are fine, now.

**Comment by hpjaj** [comment]
@kirbygit - Forest production secrets need to be removed:

```
   forest_env_secret: <%= ENV["FOREST_ENV_SECRET"] %>
   forest_auth_secret: <%= ENV["FOREST_AUTH_SECRET"] %>
```

**Pull request merged**